baseframe/forms/validators.py

```diff
@@ -161,6 +162,30 @@ def compare(self, value, other):
         return value != other
 
 
+class PublicEmailDomain(object):
+    def __init__(self, message=None):
+        self.message = message or _(u'The domain "{domain}" is not a public email domain.')
+
+    def __call__(self, form, field):
+        sniffedmx = mxsniff(field.data)
```

Comment:
I think the flag to look up the domain cache is False by default. If so, make it True.

Reply:
`cache` seems to be expecting a dict? Is the client app supposed to maintain the cache list? https://github.com/jace/mxsniff/blob/master/mxsniff/__init__.py#L185